File: ruby/scripts/hello_loop.rb
# frozen_string_literal: true

world_list = ["a", "b", "c"]

world_list.each do |world|
  puts "world is #{world}!"
end

world_hash = {"a" => "aaa", "b" => "bbb", "c" => "ccc"}

world_hash.each do |key, value|
  puts "world #{key} is #{value}!"
end

File: python/scripts/hello_loop.py
"""
Say hello.
"""

__author__ = "RyoWakabayashi <gfdmkm573@gmail.com>"
__version__ = "1.0.0"
__date__ = "2021/04/20"

world_list = ["a", "b", "c"]

for world in world_list:
    print(f"world is {world}!")

world_dict = {"a": "aaa", "b": "bbb", "c": "ccc"}

for key, value in world_dict.items():
    print(f"world {key} is {value}!")
